tests/vmi_hook_sidecar_test.go

```diff
@@ -258,3 +282,21 @@ func RenderInvalidSMBiosSidecar() map[string]string {
 		"smbios.vm.kubevirt.io/baseBoardManufacturer": "Radical Edward",
 	}
 }
+
+func RenderSidecarWithConfigMap(version string) map[string]string {
+	return map[string]string{
+		"hooks.kubevirt.io/hookSidecars": fmt.Sprintf(`[{"args": ["--version", "%s"], "image":"%s/%s:%s", "configMap": {"name": "%s","key": "%s", "hookPath": "/usr/bin/onDefineDomain"}}]`,
+			version, flags.KubeVirtUtilityRepoPrefix, sidecarShimImage, flags.KubeVirtUtilityVersionTag, configMapName, configMapKey),
+	}
+}
+
+func RenderConfigMap() *k8sv1.ConfigMap {
+	return &k8sv1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: configMapName,
```

Comment:
Nit: Using a random name would be more future-proof. You want to make sure another test would not use this configmap by mistake. Also, there is no good reason to have the name shared.

Reply:
Makes sense. I've made the change. Please lgtm again if CI's not grumpy.

Reply:
Unfortunately, this is more or less equivalent.
What you want to do is use `GenerateName` instead `Name`. You then can use the name returned by `Create`

Reply:
Ack. I looked around in the repo for how we create random names for VMs and found this approach being used. I'll update with `GenerateName`. Thanks. 👍🏽

Reply:
Addressed. Can you PTAL?